### ISSUE TITLE
Scheduling Transactions

### ISSUE THREAD

**Issue by martindale** [bitpay/copay#5293]
It'd be convenient if transactions could be scheduled, to be broadcast by whichever co-signing peer is online first after the scheduled broadcast date.  Useful for monthly bills.  See also #5292.

**Comment by BGalliotti** [comment]
This issue has been added to our Feature Request List and we will be reviewing it.
Thanks for your contribution.

**Comment by martindale** [comment]
How's the review going? :upside_down_face:

**Comment by corn91791** [comment]
Yeah so fucking funny Eric I’m gonna get you I swear to God I am I’m gonna
fucking get you bro that’s not last thing I do trust me
